### PR TITLE
Rename "rush check-versions"

### PR DESCRIPTION
**Pull request by pgonzal**
This PR shortens the "check-versions" name so that it is (1) easier to type and (2) doesn't screw up the formatting for the command line docs.  It also reduces confusion between the two checks.

- "rush check" --> "rush require" (applies to a single project, and is a heuristic)
- "rush check-versions" --> "rush check"  (applies to all projects, and is an enforcement mechanism)

### CODE REVIEW

File: apps/rush/src/actions/RequireAction.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import { CommandLineAction } from '@microsoft/ts-command-line';
+
+import RushCommandLineParser from './RushCommandLineParser';
+
+export default class RequireAction extends CommandLineAction {
+  private _parser: RushCommandLineParser;
+
+  constructor(parser: RushCommandLineParser) {
+    super({
+      actionVerb: 'require',
```
Comment (nickpape-msft) [inline apps/rush/src/actions/RequireAction.ts]: >require [](start = 19, length = 7)

I don't like this name. Other than that the PR is OK

**Comment by nickpape-msft** [comment]
:shipit:

File: apps/rush/src/actions/RequireAction.ts (hunk 1 above)
Comment (nickpape-msft) [inline apps/rush/src/actions/RequireAction.ts]: What about "check-imports" ????

---
In reply to: [116320505](https://github.com/Microsoft/web-build-tools/pull/183#discussion_r116320505) [](ancestors = 116320505)

**Comment by pgonzal** [comment]
:shipit:

**Pull request merged**